third_party/pcsc-lite/naclport/server_clients_management/src/permissions_checking/user-prompt-dialog.html

```diff
@@ -34,8 +34,6 @@
 <body>
   <h1 id="header"></h1>
 
-  <button id="close-window">×</button>
```

Comment:
The changes related to the "close" button should better be extracted into a separate PR, I think.

Also we still need this button in the app PACKAGING mode, so we shouldn't just delete it. (A probable solution could be to check the PACKAGING mode in the JS code and make the button invisible when the mode is !=app.)